policyengine_us/variables/gov/ssa/ssi/eligibility/income/ssi_income_eligible.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class ssi_income_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "Income less than the SGA limit"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        income = person("ssi_earned_income", period)
```

Comment:
just a note for the future that we'll want to deduct impairment-related work expenses https://github.com/PolicyEngine/policyengine-us/issues/3669

no need to address here